src/net.cpp

```diff
@@ -2065,6 +2066,22 @@ void CConnman::ThreadMessageHandler()
             if (flagInterruptMsgProc)
                 return;
         }
+        int64_t nNow = GetTimeMicros();
+        for (std::map<uint256, int64_t>::const_iterator tx = tx_to_ask.begin(); tx != tx_to_ask.end(); ++tx) {
+            if (tx->second > nNow)
+                continue;
+
+            std::multimap<int, std::pair<CNode*, CInv>> peersToAskTx = tx_to_ask_from.find(tx->first)->second;
+            std::multimap<int, std::pair<CNode*, CInv>>::iterator peerToAsk = peersToAskTx.begin();
+            if (peerToAsk != peersToAskTx.end()) {
+                peerToAsk->second.first->scheduled_asks.insert(peerToAsk->second.second);
```

Comment:
I find this block and especially this line barely readable. All those `peerToAsk->second.first`...
Maybe you can at least consider extracting some variables and giving them proper names?

For example `tx->first` is `tx_hash`. 
`tx-second` is a `timeout` and so on.